modules/taxonomy/animal_type/config/install/taxonomy.vocabulary.animal_type.yml

```diff
@@ -4,7 +4,8 @@ dependencies:
   enforced:
     module:
       - farm_animal_type
-name: Animal type
+name: 'Animal type'
 vid: animal_type
 description: 'A list of animal species/breeds.'
 weight: 0
+new_revision: false
```

Comment:
IIRC taxonomy terms have revisions, it's just that the UI doesn't fully support or show them. But in line with our other entity types, would it be good to enable revisions by default for taxonomy terms?

Perhaps this would require an update hook and should happen in a separate PR too.

Reply:
(Also curious: do term revisions capture the author of the change? AFAIK terms do not have any authorship information associated with them. That is arguably one of the most important things about revisions: knowing *who* made the change. Knowing *when* is also useful though... :thinking:)